docs/04-documentation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update components documentation directly in GitHub
+
+## Step 0: GitHub account
+
+If you don't already have a GitHub account, make sure to create one and log on with it.
+
+## Step 1: Open documentation file
+
+* Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
+* Locate the component you wish to edit. Components are located in `framework/components`
+* Open the README.md file in that component (example: `framework/components/ecl-blockquotes/README.md`)
+
+## Step 2: Edit the documentation
+
+* Click on the edit button (pencil icon on top right)
+* Make any modification you wish by using markdown editor. Here is a [cheatsheet for markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+* You can see result directly by switching to the "Preview" tab
+
+## Step 3: Send your modifications
+
+* Once you have modified the file, it should be sent to us. To do so, still on the edit page, scroll down to the "Commit changes" form
+* Optionally, enter a description for your modifications.
+* Make sure that the "Create a new branch" option is selected. You can keep default name for the branch.
+* Click on "Propose file change". You are directed to a page to open a pull request.
+* Just click on "Create pull request", no need to worry about the title or description.
```

Comment:
Actually, it would be helpful to already mention the [github contribution guide](https://ec-europa.github.io/europa-component-library/docs/conventions/git) and point that title and description should follow some conventions. Otherwise, `no need to worry` will turn to confusion when the person is surely corrected by us afterwards. The message should be that these are optional and helpful, but tagging, title and description are obligatory before the PR is merged. Last, but not least, the person should anticipate that we are going to update his contribution further, as it the [markdown linting task](https://ec-europa.github.io/europa-component-library/docs/conventions/markdown) will trigger only when the changes enter the pull request.